projects/bp-gallery/src/types/helper.ts

```diff
@@ -0,0 +1 @@
+export type MutationFunction<T extends (...args: never[]) => [unknown, unknown]> = ReturnType<T>[0];
```

Comment:
file can be completely removed if you convert exhibition-helper.ts into a hook as described

Reply:
exhibition-helper can't be a hook, since I have to call it inside a function